tests/unittests/delegated_operators_tests.py

```
@@ -710,20 +711,22 @@
         dataset.save()
 
-        # Execute queued operation after saving the new dataset name
+        # Execute scheduled operation after saving the new dataset name
         try:
-            self.svc.execute_queued_operations(delegation_target="test_target")
+            self.svc.execute_scheduled_operations(
+                delegation_target="test_target"
+            )
             doc = self.svc.get(doc_id=doc.id)
             self.assertEqual(doc.run_state, ExecutionRunState.COMPLETED)
         except:
```

Ruff

Do not use bare `except`

(E722)